cpp/src/round/round.cu

```diff
@@ -252,13 +253,24 @@ std::unique_ptr<column> round_with(column_view const& input,
     result_type, input.size(), copy_bitmask(input, stream, mr), input.null_count(), stream, mr);
 
   auto out_view = result->mutable_view();
-  Type const n  = std::pow(10, std::abs(decimal_places + input.type().scale()));
 
-  thrust::transform(rmm::exec_policy(stream),
-                    input.begin<Type>(),
-                    input.end<Type>(),
-                    out_view.begin<Type>(),
-                    FixedPointRoundFunctor{n});
+  constexpr int max_precision = cuda::std::numeric_limits<Type>::digits10;
```

Comment:
Does this need to be stored as a variable when it is only used in one place?

Reply:
No, it doesn't. I removed it.